src/components/Auth/Auth.tsx

```diff
@@ -52,6 +52,7 @@ export interface Props {
   view?: ViewType
   redirectTo?: RedirectTo
   onlyThirdPartyProviders?: boolean
+  withMagicLink?: boolean
```

Comment:
shall we just call it `magicLink` ?

Reply:
Sure! I have changed it. Please do let me know of any other changes 😃 .